pkg/config/v2/config.go

```diff
@@ -41,6 +42,14 @@ type MOSNConfig struct {
 	Debug               PProfConfig     `json:"pprof,omitempty"`
 	Pid                 string          `json:"pid,omitempty"`    // pid file
 	Plugin              PluginConfig    `json:"plugin,omitempty"` // plugin config
+
+	Extend []ExtendItem `json:"extend,omitempty"` // service registry auto extend config
```

Comment:
fmt

Reply:
comment

Reply:
this is already go fmted, or the linter will put a warning here